src/api/index.ts

```diff
@@ -21,10 +21,11 @@ export const getAllAccounts = async () => {
 }
 
 // ether balance, not ETH tokens
-export const getETHBalance = async () => {
+export const getETHBalance = async (account?: Account) => {
```

Comment:
`getETHBalance` doesn't exist for me on the `web3` object. Is it not `web3.eth.getBalance(...)`?